layouts/partials/head.html

```diff
@@ -0,0 +1,150 @@
+<head>
```

Comment:
What is customised here?

Reply:
The site is using blowifish icon for the tab. I changed that a little bit to add eosc icon with EVERSE colors. 

<img width="111" alt="image" src="https://github.com/graeme-a-stewart/EVERSE-ResearchSoftware.github.io/assets/44325851/c2f3d9ff-b46b-49f0-9c80-5891ffaf8769">

<img width="91" alt="image" src="https://github.com/graeme-a-stewart/EVERSE-ResearchSoftware.github.io/assets/44325851/204dae09-551f-4c14-a19d-f0d133caac0a">